docs/src/main/paradox/discovery/kubernetes.md

```diff
@@ -1,31 +1,30 @@
 ## Kubernetes API
 
-Another discovery implementation provided is one that uses the Kubernetes API. Instead of doing a DNS lookup,
-it sends a request to the Kubernetes service API to find the other pods. This method allows you to define health
-and readiness checks that don't affect the discovery method. Configuration options are provided to adjust
-the namespace, label selector, and port name that are used in the pod selection process.
+The typical way to consume a service in Kubernetes is to discover it through DNS: this will take into account liveness/readiness checks, and depending on the configuration take care of load balancing (removing the need for client-side load balancing). For this reason, for general usage the [`akka-dns`](https://doc.akka.io/docs/akka/current/discovery/index.html#discovery-method-dns) implementation is usually a better fit for discovering services in Kubernetes. However, in some cases, such as for @ref[Cluster Bootstrap](../bootstrap/index.md), it is desirable to connect to the pods directly, bypassing any liveness/readiness checks or load balancing. For such situations we provide a discovery implementation that uses the Kubernetes API.
 
 ### Dependencies and usage
 
-Kubernetes currently ignores all fields apart from service name. This is expected to change.
-
-Using `akka-discovery-kubernetes-api` is very simple, as you simply need to depend on the library::
+First, add the dependency on the component:
 
 @@dependency[sbt,Gradle,Maven] {
   group="com.lightbend.akka.discovery"
   artifact="akka-discovery-kubernetes-api_2.12"
   version="$version$"
 }
 
-And configure it to be used as default discovery implementation in your `application.conf`:
+As described above, it is uncommon to use the Kubernetes API discovery
+mechanism as your default discovery mechanism. When using it with Akka Cluster
+Bootstrap, it is sufficient to configure it as described @ref[here](../bootstrap/kubernetes-api.md).
+To load it manually, use `loadServiceDiscovery` on the `Discovery` extension:
```

Comment:
To further highlight that it's not needed for bootstrap: "Otherwise, to load it manually..."
?